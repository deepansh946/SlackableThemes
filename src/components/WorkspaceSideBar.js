import React from 'react';

import styles from "../styles/WorkspaceSideBar.modules.css";

const WorkspaceSideBar = () => {
    const workspaces = ["workspaceOne", "workspaceTwo", "workspaceThree"]

    return (
        <div className="workspaceSideBar">
            <ul className="workspaceList">
                {workspaces.map(workspace => <li key={workspace}>{workspace}</li>)}
            </ul>
        </div>
    )
}

export default WorkspaceSideBar;