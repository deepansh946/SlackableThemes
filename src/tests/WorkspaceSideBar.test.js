import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import WorkspaceSideBar from '../components/WorkspaceSideBar';


Enzyme.configure({ adapter: new Adapter() });

describe("WorkspaceSideBar", () => {
    it("should render workspace side bar", () => {
        const { container } = shallow(<WorkspaceSideBar/>);
        expect(container).toMatchSnapshot();
    })

})

