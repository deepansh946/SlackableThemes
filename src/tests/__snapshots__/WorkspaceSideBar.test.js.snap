// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkspaceSideBar should render workspace side bar 1`] = `undefined`;
